addons/map/functions/fnc_switchFlashlight.sqf

```diff
@@ -16,10 +16,18 @@
 
 #include "script_component.hpp"
 
-params ["_flashlight"];
+params ["_newFlashlight"];
+private _oldFlashlight = GVAR(flashlightInUse);
```

Comment:
I figured since it would never be globally known, it wouldn't be needed. Would it be worthwhile to make it a PVAR?